packages/core/src/integrations/requestdata.ts

```diff
@@ -45,6 +45,7 @@ const _requestDataIntegration = ((options: RequestDataIntegrationOptions = {}) =
 
       const includeWithDefaultPiiApplied: RequestDataIncludeOptions = {
         ...include,
+        // todo: also exclude cookies and headers if sendDefaultPii is false
```

Comment:
This is actually something we should do - the question is if we do this in a minor or major?

Reply:
In the Http Client integration for browser, we only include the cookie data if this is explicitly enabled:

https://github.com/getsentry/sentry-javascript/blob/63476fa8931acffb9e73fdafee85f4f939cf1c1e/packages/browser/src/integrations/httpclient.ts#L140-L146

Reply:
Agreed, we should do it but my gut feeling is to do it in a major (see review comment)